Tests/SonarQube.Common.UnitTests/ProcessRunnerTests.cs

```diff
@@ -226,7 +226,10 @@ public void ProcRunner_ArgumentQuoting()
                 "\"quoted\"",
                 "\"quoted with spaces\"",
                 "/test:\"quoted arg\"",
-                "unquoted with spaces" };
+                "unquoted with spaces",
+                "quote in \"the middle",
+                "quotes & ampersands",
```

Comment:
Is this test string supposed to have a quote in it? *quotes & ampersnads* suggests it should.